pokeclicker/pokeclicker
# Fix farmDisplay collapse contents breaking after disabling module

Fixed disabling a collapsible module breaking DisplayObservables' tracking of the collapse's state. This mainly affected the farmDisplay, breaking its contents if it was disabled while collapsed or breaking the performance-saving bindings if it was disabled while open.

## How Has This Been Tested?
1. Collapse `#plotListCard`
2. Disable the showFarmModule setting
3. Enable the showFarmModule setting

On develop this consistently breaks the contents of the farm module as DisplayObservables thinks it's still hidden. Works as expected after these changes.
